classes/Database.php

```diff
@@ -0,0 +1,25 @@
+<?php
+class Database{
+ 
+    // specify your own database credentials
+    private $host = "127.0.0.1";
+    private $db_name = "test_schema";
+    private $username = "root";
+    private $password = "";
+    public $conn;
+ 
+    // get the database connection
+    public function getConnection() {
+ 
+        $this->conn = null;
```

Comment:
1) Problema cu metoda getConnection() din aceasta clasa este ca defiecare data cand ii ceri conexiunea, creaza cate o conexiune noua.
2) Apelul $this->conn = null este redundant, deoarece orice proprietate declarata intr-o clasa are la inceput valoarea null daca nu a fost initializata prin declaratie.
3) Cred ca proprietatea $conn trebuia sa fie privata.